modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -380,18 +424,53 @@ export default class ParticipantConnectionStatus {
             isConnActiveByJvb = true;
         }
 
-        const isConnectionActive
-            = isConnActiveByJvb
-                && (isVideoMuted || (isInLastN && !isVideoTrackFrozen));
+        let newState = ParticipantConnectionStatus.INACTIVE;
+
+        if (isConnActiveByJvb) {
+            if (isInLastN) {
```

Comment:
I still find it valid that it could happen that there will be some timing issues with "lastN" events and it may happen we display working video as inactive, but maybe let's see how it behaves in practice.